package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git+https://github.com/DoSomething/gambit.git"
   },
   "scripts": {
-    "test": "NODE_ENV=test ava --serial",
+    "test": "NODE_ENV=test ava --serial --fail-fast",
```

Comment:
Happy to remove this, but I slightly prefer it?  Could be swayed though.

Reply:
What about adding another script named `test-fast`? I believe there is something to gain from seeing all failed tests across different files.

Reply:
Good call! Will update.